consensus/src/simplex/actors/voter/round.rs

```diff
@@ -141,6 +141,11 @@ impl<S: Scheme, D: Digest> Round<S, D> {
         self.certify_handle = None;
     }
 
+    /// Returns true if certification is pending (has notarization, not yet certified, no handle).
+    pub const fn is_certification_pending(&self) -> bool {
```

Comment:
update should_certify to indicate whether or not pending instead of requiring another method here.